dockerfiles/alpine/analytics/dashboard/docker-entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```

Comment:
@Vathsan shall we update the license header year since this file has been introduced in this PR? Please do the same for other, similar files.

Reply:
Fixed in 532f659e7ec08c156c18acfddc9e437b8efc431e